src/utils/DragAndDrop.js

```diff
@@ -181,6 +182,68 @@ define(function (require, exports, module) {
             var pathList = [];
             var errorList = [];
 
+            function handleRegularFile(deferred, file, filename, buffer, encoding) {
+                file.write(buffer, {encoding: encoding}, function(err) {
+                    if (err) {
+                        deferred.reject(err);
+                        return;
+                    }
+
+                    if(encoding === "utf8") {
```

Comment:
So that I don't try to open a non-UTF8 file in the editor (e.g., the .zip file).